wqu_app/message.py

```diff
@@ -13,9 +13,13 @@ def get_geolocation(ip_address):
     response = requests.get(f'https://ipinfo.io/{ip_address}')
     data = response.json()
     city = data['city']
-    coords = [float(coord) for coord in data['loc'].split(',')]
     region = data['region']
-    return coords, city, region
+    coords = [float(coord) for coord in data['loc'].split(',')]
+    if region == None or city not in region:
+    	return coords, city
+    else:
+    	city = city + ", " + region
+    return coords, city
```

Comment:
This looks a lot cleaner and simpler. A nit pick, it's Pythonic to write `region is None` as opposed to `region == None`. See here: https://www.python.org/dev/peps/pep-0008/#programming-recommendations